### PR TITLE
Add transform properties to VideoSource node

### PR DESCRIPTION
**Pull request by DmitrySamoylov**
Story details: https://app.clubhouse.io/lumeo/story/3544

### CODE REVIEW

**Review by jplatte — APPROVED** [review]

File: pipeline/src/node_properties/video_source_properties.rs [hunk 1]
```diff
@@ -26,6 +29,37 @@ pub struct CommonVideoSourceProperties {
...
+    /// Crop video source.
+    /// Format: "left:right:top:bottom".
+    pub crop: Option<Crop>,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum RotateDirection {
+    /// Rotate clockwise by 90 degrees.
+    /// Changes resolution, for example 640x480 becomes 480x600.
```
Comment (SIGLOST) [inline pipeline/src/node_properties/video_source_properties.rs:53]: on the comment I guess you mean 480x640  :smiley:  (also on L61)

**Review by SIGLOST — APPROVED** [review]
lgtm, minor comment

File: pipeline/src/node_properties/video_source_properties.rs [hunk 2]
```diff
@@ -26,6 +29,37 @@ pub struct CommonVideoSourceProperties {
...
+    Cw90,
```
Comment (zeenix) [inline pipeline/src/node_properties/video_source_properties.rs:54]: I'd very much prefer not using unknown abbreviations: `CounterClock90` etc.

**Review by zeenix — CHANGES_REQUESTED** [review]
LGTM otherwise.

**Review by zeenix — APPROVED** [review]

**Pull request merged**